bin/utils/convert-argv.js

```diff
@@ -137,11 +136,20 @@ module.exports = function(...args) {
 
 	function processConfiguredOptions(options) {
 		if (options) {
-			const webpackConfigurationValidationErrors = validateSchema(webpackConfigurationSchema, options);
-			if (webpackConfigurationValidationErrors.length) {
-				const error = new WebpackOptionsValidationError(webpackConfigurationValidationErrors);
-				console.error(error.message, `\nReceived: ${typeof options} : ${JSON.stringify(options, null, 2)}`);
-				process.exit(-1); // eslint-disable-line
+			let error;
```

Comment:
Could you make a function out of it and export it? So we can test it and make sure 100% that it works.

Reply:
yup will do